### PR TITLE
Add more tags to Github.

### PR DESCRIPTION
**Pull request by jmchilton**
I spent the last two days going through issues and adding tags and I thought these ones would have been helpful/meaningful/actionable in some way.

### CODE REVIEW

File: doc/source/project/issues.rst [hunk 1]
```diff
@@ -122,10 +138,12 @@ particular domain, as well as more organized release notes.
 - ``area/histories``
 - ``area/i18n`` - Internationalization and localization
 - ``area/jobs``
+- ``area/jobs/kubernetes``
 - ``area/libraries`` - Change related to data libraries
 - ``area/objectstore``
 - ``area/performance``
-- ``area/reports``
+- ``area/reports`` - The reports webapp
+- ``area/rules`` - Rule builder
```
Comment (nsoranzo) [inline doc/source/project/issues.rst:146]: Maybe this could be a subtopic of `area/UI-UX` ?

File: doc/source/project/issues.rst [hunk 2]
```diff
@@ -97,6 +97,22 @@ These sorts of terminal states *and their justifications* (e.g. the
...
+- ``status/waiting/response`` - this issue or pull request is waiting for
+  a response from the author. The issue or pull request may be assumed stale
+  and closed after a month. The committers reserve the right to close issues
+  and pull requests without this process, but this tag makes tracking explicit
+  and easy.
+- ``status/waiting/deployment/org`` - this issue was exhibited on usegalaxy.org
+  and committers believe a fix has been committed but are not certain and so the
+  issue should be rechecked after deployment of the target branch. Typically the
+  committers will close out issues when a fix has been merged, but in cases they
+  are uncertain the fix will fix the underlying issue this tag may be used.
```
Comment (nsoranzo) [inline doc/source/project/issues.rst]:
```suggestion
  are uncertain whether the fix will correct the underlying issue, this tag may be used.
```

File: doc/source/project/issues.rst (hunk 1 above)
Comment (jmchilton) [reply doc/source/project/issues.rst:146]: I considered this and would have done it, I just didn't know if area/UI-UX made more sense or area/client as a base. I generally use area/UI-UX but I don't know that it is a consensus. I'm happy for more structure and more descriptions if someone wants to follow up.

Comment (nsoranzo) [reply doc/source/project/issues.rst:146]: I'm not sure what's the difference between `area/client` and `area/UI-UX`, actually `area/client` is not even listed here.

**Pull request merged**